<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="FFTTab">
       <attribute name="title">
        <string>FFT</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCustomPlot" name="FFTCustomPlot" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FreqTab">
       <attribute name="title">
        <string>Fixed Frequency</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="startFreq">
          <item>
           <widget class="QLabel" name="labelStart">
            <property name="text">
             <string>Start frequency (Hz):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxStart">
            <property name="maximum">
             <number>63</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="endFreq">
          <item>
           <widget class="QLabel" name="labelEnd">
            <property name="text">
             <string>End frequency (Hz):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxEnd">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>64</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCustomPlot" name="FreqCustomPlot" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="logScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>408</width>
             <height>355</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextBrowser" name="log">
              <property name="minimumSize">
               <size>
                <width>390</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>390</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="sideMenu">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="gameLabel">
        <property name="text">
         <string>Game: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="gameComboBox">
        <item>
         <property name="text">
          <string>Pong</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tetris</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="StartGameButton">
        <property name="text">
         <string>Start Game</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="channelLabel">
        <property name="text">
         <string>Channel:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="channelComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="startAcqButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="StopAcqButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_configuration_as"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_window"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionShow_normal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDisplay"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open configuration</string>
   </property>
  </action>
  <action name="actionSave_configuration_as">
   <property name="text">
    <string>Save configuration as</string>
   </property>
  </action>
  <action name="actionOpen_window">
   <property name="text">
    <string>Open new window</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionShow_normal">
   <property name="text">
    <string>Show normal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionFull_Screen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_normal</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
